web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx

```diff
@@ -18,19 +18,19 @@ export const ReviewLineItem = (props: Props): ReactElement => {
 
   return (
     <div
-      className={`${isTabletAndUp ? "display-flex" : "display-block"} ${
-        props.marginOverride || "margin-top-1"
-      }`}
+      className={`${isTabletAndUp ? "grid-row" : "display-block"} ${props.marginOverride || "margin-top-1"}`}
       data-testid={props.dataTestId}
     >
-      <div className="text-bold width-15rem margin-right-5">
+      <div className="text-bold grid-col flex-4">
         {props.labelContextualInfo ? (
           <ContextualInfoButton text={`${props.label}:`} id={props.labelContextualInfo} />
         ) : (
           `${props.label}:`
         )}
       </div>
-      <Content>{props.formatter ? props.formatter(props.value) : props.value}</Content>
+      <div className={"grid-col flex-8"}>
+        <Content>{props.formatter ? props.formatter(props.value) : props.value}</Content>
+      </div>
```

Comment:
This is totally me nit picking, but is there a reason we weren't able to just provide these classNames to `Content`? It looks like they'd get appended to the className list for a wrapper div in that component.

Reply:
good catch, we can just pass into content, making the change.

Reply:
good catch, we can just pass into `Content`, making the change.